controller/sessions.js

```diff
@@ -0,0 +1,83 @@
+var
+  _ = require('lodash'),
+  changeCase = require('change-case'),
+  async = require('async'),
```

Comment:
You might not want to use `async` as a variable name. It is a reserved keyword in Node 8. I think you're running Node ~4, but if c9 eventually upgrades, I think this will break.